sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -478,6 +478,7 @@ object SizeBasedWindowFunction {
 case class RowNumber() extends RowNumberLike {
   override val evaluateExpression = rowNumber
   override def sql: String = "ROW_NUMBER()"
```

Comment:
Yep. We can remove `sql` here. The default implementation of `Expression` can handle this. I will remove `sql` here. Thank you.